### PR TITLE
Simplify CRA basic chat demo (as was done for docs chat demo)

### PR DESCRIPTION
**Pull request by juberti**

### CODE REVIEW

File: packages/create-react-app-demo/src/basic-chat/ai.tsx [hunk 1]
```diff
@@ -1,63 +1,18 @@
...
         conversation back to these topics that you care so deeply about.
       </SystemMessage>
-      {conversation.map((chatMessage) =>
-        chatMessage.type === 'assistant' ? (
-          <AssistantMessage>{chatMessage.content}</AssistantMessage>
-        ) : (
-          <UserMessage>{chatMessage.content}</UserMessage>
-        )
+      {conversation.map((message, index) =>
+        index % 2 != 0 ? <AssistantMessage>{message}</AssistantMessage> : <UserMessage>{message}</UserMessage>
```
Comment (NickHeiner) [inline packages/create-react-app-demo/src/basic-chat/ai.tsx:14]:
```suggestion
        index % 2 ? <AssistantMessage>{message}</AssistantMessage> : <UserMessage>{message}</UserMessage>
```

File: packages/create-react-app-demo/src/basic-chat/ai.tsx
Comment (NickHeiner) [inline packages/create-react-app-demo/src/basic-chat/ai.tsx:1]: Now that this file is so small, I'd just merge its contents with `index.tsx`. It'll better reinforce the idea that we have seamless integration between AI and UI if we don't have separate files called `ai`. :smile:

File: packages/create-react-app-demo/src/basic-chat/index.tsx [hunk 2]
```diff
@@ -1,46 +1,58 @@
-import React from 'react';
-import { AIRoot, ChatMessage, conversationAtom, modelCallInProgress } from './ai.tsx';
-import { useAtom } from 'jotai';
+/** @jsxImportSource ai-jsx/react */
+import * as AI from 'ai-jsx/react';
+import { useAI } from 'ai-jsx/react';
```
Comment (NickHeiner) [inline packages/create-react-app-demo/src/basic-chat/index.tsx:3]: You can get rid of this import and do `AI.useAI` below.

File: packages/create-react-app-demo/src/basic-chat/index.tsx [hunk 3]
```diff
@@ -59,12 +71,3 @@ function ConversationHistory() {
...
 }
-
-export function BasicChat() {
-  return (
-    <>
-      <AIRoot />
-      <ConversationHistory />
-    </>
-  );
-}
```
Comment (NickHeiner) [inline packages/create-react-app-demo/src/basic-chat/index.tsx:70]: Feels much better to get rid of this nasty nast.

**Review by NickHeiner — APPROVED** [review]

File: packages/create-react-app-demo/src/basic-chat/index.tsx (hunk 2 above)
Comment (juberti) [reply packages/create-react-app-demo/src/basic-chat/index.tsx:3]: not sure I even am using useAI, I just removed it

**Pull request merged**